project.mml

```diff
@@ -1965,7 +1966,7 @@ Layer:
             NULL AS way_area,
             CASE WHEN building = 'no' OR building IS NULL THEN 'no' ELSE 'yes' END AS is_building
           FROM planet_osm_line
-          WHERE (man_made IN ('pier', 'breakwater', 'groyne', 'embankment')
+          WHERE (man_made IN ('pier', 'breakwater', 'groyne', 'embankment', 'goods_conveyor')
```

Comment:
'goods_conveyor' is added here, but there is no mss code to render the name. Is this intentional? 

I see that there are only 78 different names for goods conveyors, and at least half of those are "band", "belt", "conveyor" or a similar description rather than a name, so perhaps we don't need to render the names of these features?

Reply:
yes, that's better to remove it